docs/modules/ROOT/pages/how-to/manage-projects-and-namespaces.adoc

```diff
@@ -0,0 +1,38 @@
+= Manage Projects and Namespaces
+
+This page explains the various mechanisms available to create OpenShift Projects in APPUiO Cloud.
+
+== Creating Projects
+
+Through the web console:: Click on the btn:[Create Project] button and fill the form to create a new project.
+
+Through the Command Line:: Use the `oc new-project` command.
+
+Through the API:: Use the `ProjectRequest` resource to create a new project.
+
+== Project Ownership
+
+OpenShift Projects will always be assigned to the default organization specified by the `appuio.io/organization` label. Check the xref:explanation/organizations-and-rbac.adoc[Organizations and RBAC] page for more information.
+
+== Namespaces
+
+Namespaces can be created in two ways:
+
+. With the command line `kubectl create namespace` (or `oc create namespace`)
+. Through the API, using the `Namespace` resource.
+
+WARNING: You can't create namespaces through the web console.
+
+Namespaces created with any of these mechanisms can be managed just like any other OpenShift projects.
+
+Just like projects, namespaces are assigned to a specific organization using the `appuio.io/organization` label. Namespaces without such label will be assigned to the default organization.
```

Comment:
For the less experienced users, it would be handy to have an example. Please add a YAML snipped for a namespace with an organization label.

Reply:
Should we also describe how one finds out about the organization name?

Reply:
Yes, we have to. See confusion in BBUL-26.

Reply:
How can a user find its default organization name?

Reply:
Added YAML snippet

Reply:
> How can a user find its default organization name?

Looks like this is not possible until the Control API is available. The best I can offer is: create a new project (some random throw away name) and look at the label. If you get an error, you have none.

Reply:
Ok, I got hinted an undocumented way to achieve this.

```
 oc get user "~" -ojsonpath="{.metadata.annotations['appuio\.io/default-organization']}"
```

Reply:
Solved